example/resnet50.yml

```diff
@@ -11,4 +11,4 @@ outputs:
   - name: "output"
     shape: [ -1, 1000 ]
     dtype: TYPE_FP32
-convert: true
\ No newline at end of file
+convert: false
```

Comment:
Is it necessary to set `convert` to false?

Reply:
It is just for convenience of debug. So True and false are both ok